app/src/main/java/com/github/se/polyfit/ui/navigation/Navigation.kt

```diff
@@ -72,4 +72,8 @@ class Navigation(private val navHostController: NavHostController) {
     Log.i("Navigation", "Navigating to $route")
     navHostController.navigate(route)
   }
+
+  fun navigateToPostInfo() {
+    navigateTo(Route.PostInfo)
+  }
```

Comment:
I'm not sure, but is it better than just calling navigateTo(Route.PostInfo)?

Reply:
I just tried to align myself with the practices of the overall codebase. I think that it looks a bit cleaner because there is no need to pass the destination as an argument.